eruditus/app_commands/ctf/__init__.py

```diff
@@ -1347,6 +1349,98 @@ async def submit(
         """
         await interaction.response.send_modal(FlagSubmissionForm(members=members))
 
+    @app_commands.command()
+    @app_commands.autocomplete(name=_challenge_autocompletion_func)
+    @_in_ctf_channel()
+    async def showsolvers(
+        self, interaction: discord.Interaction, name: Optional[str] = None
+    ) -> None:
+        """
+        Show the list of solvers for the provided/current challenge.
+        """
+
+        async def followup(content: str, ephemeral=True, **kwargs) -> None:
+            if not interaction:
+                return
+            await interaction.followup.send(content, ephemeral=ephemeral, **kwargs)
+
+        await interaction.response.defer()
+
+        if name is None:
+            challenge = get_challenge_info(thread=interaction.channel_id)
+            if challenge is None:
+                await followup(
+                    (
+                        "Run this command from within a challenge thread, "
+                        "or provide the name of the challenge you wish to stop "
+                        "working on."
+                    )
+                )
+                return
+        else:
+            challenge = get_challenge_info(name=name)
+            if challenge is None:
+                await followup("No such challenge.")
+                return
+
+        ctf = get_ctf_info(guild_category=interaction.channel.category_id)
+        ctx: PlatformCTX = PlatformCTX.from_credentials(ctf["credentials"])
+        platform = await match_platform(ctx)
+
+        if not platform:
+            await followup("Invalid URL set for this CTF, or platform isn't supported.")
+            return
+
+        solvers = []
+
+        async for solver in platform.pull_challenge_solvers(
+            ctx=ctx, challenge_id=challenge["id"], limit=0
+        ):
+            solvers.append(solver)
+
+        num_solvers = len(solvers)
+        if num_solvers == 0:
+            await followup("This challenge has not been blooded yet.")
+            return
+
+        me = await platform.get_me(ctx)
+        our_team_name: str = me.name if me is not None else TEAM_NAME
+
+        embed = None
+        num_fields = 0
+
+        for idx, solver in enumerate(solvers, start=1):
+            # If we reached Discord's maximum number of fields per
+            # embed, we send the previous one and create a new one.
```

Comment:
Maximum line length in this project is 88 chars
```suggestion
            # If we reached Discord's maximum number of fields per embed, we send the
            # previous one and create a new one.
```